apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -45,71 +26,41 @@ require('/views/shared/test/unit/mock_utils.js');
 require('/views/shared/js/activity_handler.js');
 
 var mocksHelperForActivityHandler = new MocksHelper([
+  'ActivityClient',
+  'ActivityShim',
   'Attachment',
-  'Contacts',
   'Draft',
   'Drafts',
   'MessageManager',
-  'NotificationHelper',
-  'Notify',
-  'Settings',
-  'SettingsURL',
-  'SilentSms',
-  'SMIL',
-  'Threads',
-  'Utils',
   'Navigation',
-  'ActivityShim',
-  'ActivityClient'
+  'Settings',
+  'Utils'
 ]).init();
 
 suite('ActivityHandler', function() {
   mocksHelperForActivityHandler.attachTestHelpers();
 
-  var realSetMessageHandler;
-  var realWakeLock;
-  var realMozApps;
-  var realMozL10n;
-
-  var isDocumentHidden;
+  var navigatorMocks = new Map([
```

Comment:
You mean we don't need to recover original navigator based API so we don't need ```navigator[Symbol.for('mocks')].forEach((key) => delete navigator[key]);``` in ```suiteTeardown``` at all (and then ```Symbol.for('mocks')``` magic as well :))?

If so, then I don't have any particular reason to do so, I just had a mixed feelings about it and if you think that we don't need to recover Navigator APIs in ```suiteTeardown``` I tend to agree and happily will simplify this piece of code :)